lib/metatags/Metatags.js

```diff
@@ -1,70 +1,47 @@
 "use strict";
 
-var OpenGraphTag = require("./OpenGraphTag");
-var LinkTag = require("./LinkTag");
-var StandardTag = require("./StandardTag");
-var _ = require("lodash");
+var $ = require("../application/jquery");
 
-var Metatags = function(pairs) {
-    if (!pairs) {
-        throw new Error("You cannot create empty metatags");
-    }
-
-    this.pairs = pairs;
-
-    this.tagViews = [];
-};
-
-Metatags.prototype = {
-
-    pairs: null,
-
-    tagViews: null,
+function createOpenGraphTag(property, content) {
+    return '<meta property="' + property + '" content="' + content + '" data-ephemeral="true">';
+}
 
-    createTagView: function(tagName, tagValue) {
-        if (/^og\:/.test(tagName)) {
-            return OpenGraphTag.createView(tagName, tagValue);
-        }
+function createNormalTag(name, content) {
+    return '<meta name="' + name + '" content="' + content + '" data-ephemeral="true">';
+}
 
-        if (/^canonical$/.test(tagName)) {
-            return LinkTag.createView(tagName, tagValue);
-        }
+function createLinkTag(rel, href) {
+    return '<link rel="' + rel + '" href="' + href + '" data-ephemeral="true">';
+}
 
-        return StandardTag.createView(tagName, tagValue);
-    },
-
-    createTagViews: function() {
-        if (this.tagViews && this.tagViews.length > 0) {
-            return this.tagViews;
-        }
-
-        var pairs = this.pairs;
+function createTag(name, value) {
+    if (/^og\:/.test(name)) {
+        return createOpenGraphTag(name, value);
+    }
 
-        for (var key in pairs) {
-            if (pairs.hasOwnProperty(key)) {
-                this.tagViews.push(this.createTagView(key, pairs[key]));
-            }
-        }
+    if (/^canonical$/.test(name)) {
+        return createLinkTag(name, value);
+    }
 
-        return this.tagViews;
-    },
+    return createNormalTag(name, value);
+}
 
-    closeTagViews: function() {
-        _.each(this.tagViews, function(tagView) {
-            tagView.close();
-        });
+var Metatags = function(pairs) {
+    if (!pairs) {
+        throw new Error("You must provide name-content pairs when creating Metatags");
+    }
 
-        this.tagViews = [];
-    },
+    var html = "";
 
-    sameAs: function(metatags) {
-        if (metatags && _.isEqual(metatags.pairs, this.pairs)) {
-            return true;
-        }
+    Object.keys(pairs).forEach(function(name) {
+        html += createTag(name, pairs[name]);
+    });
 
-        return false;
-    }
+    this.$html = $(html);
```

Comment:
No, the slowest part would be appending this to the head, but it is nearly 100% faster than the original solution.